### PR TITLE
Merge additional receivers with the common receivers

### PR DESCRIPTION
**Pull request by suhaibmujahid**
With this PR, tools can declare `additional_receivers` in their configurations. The list of receivers in the `additional_receivers` field will be merged with the common receivers. If a tool declares the `receivers` field, it will replace the common receivers.

### CODE REVIEW

File: auto_nag/utils.py [hunk 1]
```diff
@@ -74,6 +74,20 @@ def get_config(name, entry, default=None):
     return tool_conf.get(entry, default)
 
 
+def get_receivers(tool_name):
+    receiver_lists = get_config("common", "receiver_list", default={})
+
+    receivers = get_config(tool_name, "receivers", [])
+    if isinstance(receivers, six.string_types):
```
Comment (marco-c) [inline auto_nag/utils.py:81]: Nit: we no longer need to use six given that we only support Python 3 now, we can just go with `isinstance(receivers, str)`

**Review by marco-c — DISMISSED** [review]

File: auto_nag/utils.py (hunk 1 above)
Comment (suhaibmujahid) [reply auto_nag/utils.py]: Done in e757368caa42905ca19890dc4f87f2cf118dfc8c

**Review by marco-c — APPROVED** [review]
LGTM, @calixteman do you agree with these changes?

**Review by calixteman — APPROVED** [review]
Better like this, thank you.

**Pull request merged**